components/logins/android/src/main/java/mozilla/appservices/logins/LoginsStorage.kt

```diff
@@ -12,50 +12,119 @@ class SyncUnlockInfo (
 )
 
 interface LoginsStorage : AutoCloseable {
-
+    /**
+     * Lock (close) the database.
+     *
+     * @throws [MismatchedLockException] if the database is already locked
```

Comment:
should there be a LoginsStorageException here as well?

Reply:
Nope!

Reply:
Ok 👍